ministere-sante/2020-03-04.yaml

```diff
@@ -0,0 +1,14 @@
+date: 2020-03-04
+source:
+  nom: Ministère des Solidarités et de la Santé
+  url: https://www.dailymotion.com/video/x7siev3
+  archive: http://web.archive.org/web/20200320130306/https://www.dailymotion.com/video/x7siev3
```

Comment:
Je suis en train de tester ce lien. Mais la vidéo ne fonctionne pas chez moi. Donc à supprimer.

@cecilaki : la vidéo fonctionne chez toi ? :)

Reply:
@ColinMaudry : merci de tester tous les liens :wink:

Reply:
Oui chez moi ça fonctionne bien

Reply:
@cecilaki : merci de ta réponse, je vais hard try dans ce cas :wink:

Reply:
@cecilaki: désolé d'insister. tu as bien testé le lien noté dans `archive` ?
 A savoir : http://web.archive.org/web/20200320130306/https://www.dailymotion.com/video/x7siev3

Chez moi, ça donne ça:
![image](https://user-images.githubusercontent.com/6597721/77193070-ed504980-6add-11ea-85e7-5536cb1e227a.png)

Reply:
Ah oui mais du coup on a décidé de supprimer le lien d'archive

Reply:
ça roule, je pense que archive.org ne sait pas sauvegarder les vidéos. en même temps, ça ferais tellement trop de données à sauvegarder, je vais purger les autres archive :+1:

Reply:
Top, oui effectivement c'est mieux !